streetcrm/admin.py

```diff
@@ -791,6 +791,17 @@ class ParticipantAdmin(mixins.AdminArchiveMixin, mixins.SignInSheetAdminMixin, S
     )
     import_path = reverse_lazy("import-participants")
 
+    export_header = [
```

Comment:
So if we add more fields to Participant, we'll have to remember to come here and update this list, right?

If so, could we at least add a comment wherever Participant is defined in the model, mentioning this, and a cross-referencing comment here.

Reply:
Well, yes and no.  This isn't the full list of everything that's actually in participant.  It's the list of what was in the export when you do a search for participants (for instance, city is omitted).  Maybe we should step back and do a clarification of what should be in the export?

Reply:
What the users requested was that when exporting Participants, we export everything about them, including even the internal ID.  This makes sense to me -- they can always ignore a column in the CSV if they want to, but they can't add a column that's not there.  As stated in issue #343: "Include all fields per participant, not just the fields that happen to be displayed on the Participants page."

Reply:
Updated!  With the comment added to the model file